### ISSUE TITLE
Improve citation ordering

### ISSUE THREAD

**Issue by taiya** [cvpr-org/author-kit#7]
Removed this comment from end of `\subsection{Miscellaneous}` subsection of the author-kit, and converted it into an issue

```
% Update the cvpr.cls to do the following automatically.
% For this citation style, keep multiple citations in numerical (not
% chronological) order, so prefer \cite{Alpher03,Alpher02,Authors14} to
% \cite{Alpher02,Alpher03,Authors14}.
```

**Comment by adinriv** [comment]
The order of the bibliography depends on the style we use.  If we want to use order in appearance vs. chronological, for example.  We could use IEEEtranN for the unsorted and with natbib support.

I prefer to have the group of citations sorted, since it helps checking the group.  And with the numerical styles have them in order of appearance.  But that is a style preference.

Regarding the support for the bibliography and citations, we should consider porting the whole system to biblatex instead of bibtex as well.  But that is better for another issue.

**Comment by taiya** [comment]
I think biblatex we can do for next year.
Looks to be a larger change which requires much more work.
(configuring all the options to have style parity)

I had started, but was taking me too long.

**Comment by taiya** [comment]
@sroth-visinf to comment on this.

**Comment by sroth-visinf** [comment]
I agree with @tayia here. I had also wanted to switch to BibLaTeX already in 2022, but adjusting the formatting of the entries so that they conform to the usual style proved rather tedious. I had started but found that there are loads of corner cases. Plus this needs to be thoroughly tested before it is rolled out. CVPR/ICCV/etc. program chairs cannot possibly provide LaTeX support to sort out people's issues. Aiming to do this next year with sufficient lead time is probably best.

**Comment by adinriv** [comment]
If you can provide a list of the test entries, I can work on them (depending on how troublesome they are to iron out).  

Also, since the base biblatex style comes from IEEE, my guess is that it should be less problematic unless there have been style choices made along the way.

In the attempt to port (#10) you can see what has been ported and how it is working so far.
